.vscode/launch.json

```diff
@@ -0,0 +1,28 @@
+{
+   // Use IntelliSense to find out which attributes exist for C# debugging
```

Comment:
Are comments like this okay in JSON, github highlights them red

Reply:
Tried to comment on this using my phone, but this text box doesn't work in MobileSafari. This way fine the way it is, in fact, it came that way. The reason is is that this isn't true JSON, but instead a VS Code-flavored variant that supports comments. The comments actually came from the default launch.json.